brave/brave-core
# fix(wallet): Allow Creating Wallet on Buy Screen

## Description 

Fixes a bug where you were not able to create an `Account` on the new `Buy` screen.


Resolves <https://github.com/brave/brave-browser/issues/42081>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Create a brand new Wallet and then navigate to the `Buy` screen
2. Select `ZEC` as the token to buy.
3. You will be prompt to create an account.
4. If you click `Create Account` it should auto select that `Account`
5. If you click `Cancel` it should close the modal and remain on the previous selected `Token`

https://github.com/user-attachments/assets/6a465c98-fc03-4ad7-a49c-f5ac4fcd3302
